### ISSUE TITLE
Dropping Ruler in Full Screen Makes Donk Noise

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/gravity-force-lab#254]
**Device**
Jordan
**Browser**
Safari
**OS**
Mac 10.14
**Problem Description**
For https://github.com/phetsims/QA/issues/480.
When safari is in full screen, if you use keyboard nav to "drop" the ruler it makes the "you can't do that" donk sound. It does not do this when picking up the ruler. It still does drop the ruler. Similar to https://github.com/phetsims/molarity/issues/210
**Steps to Reproduce**
1. Mute sim with nav bar icon
2. Go to full screen
3. Use keyboard nav to pick up ruler
4. Use keyboard nav to drop ruler

**Comment by zepumph** [comment]
https://github.com/phetsims/gravity-force-lab/issues/256 is making this hard to test. I will take a look over there before handling this one.

**Comment by zepumph** [comment]
Maybe it is will look something like this in releaseDraggable:

`      event && event.domEvent && event.domEvent.preventDefault();`

**Comment by zepumph** [comment]
After discussing with @jessegreenberg, and reviewing the last comment https://github.com/phetsims/molarity/issues/210#issuecomment-584412281 which caused the 24th bug in https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/edit, I think that there is likely nothing to be done here.

>Use keyboard nav to drop ruler

I'm interested if you get this when pressing escape and enter/spacebar. From that other issue it is likely that you won't be able to fix this for enter/space, but perhaps for the escape button. I will make a commit as a test (since i have no way to test locally), and then we can go from there.

**Comment by zepumph** [comment]
I added a potential fix above, but it is very much prototype code. I did this by adding `preventDefault` to the events on release. This didn't work for enter though, when I did this, it took two enter presses to release. I would like some pretty specific answers to some questions to inform how I will go about fixing (or leaving) the code alone.

First, you said "Use keyboard nav to drop ruler". There are currently 5 ways to drop the ruler. Please tell me each of the ways that the donk occurs in the rc version:
* When grabbed, press space
* When grabbed, press enter
* When grabbed, press escape
* When grabbed, press tab/shift tab
* Using a mouse, clicking the mouse and then releasing it (same grab/drag code running in the background)

Next please let me know each of the 5 above that are fixed on master, and used to have a donk but don't now.

Thank you!

**Comment by KatieWoe** [comment]
Only happens with space. Hard to tell with escape as it exits Full Screen rather than dropping the ruler, but no donk sound happens.

**Comment by zepumph** [comment]
Does this mean it still happens with space on master?

**Comment by KatieWoe** [comment]
Sorry, I thought you were asking for clarification on rc.2. Let me check on master.

**Comment by zepumph** [comment]
I was hoping for both in https://github.com/phetsims/gravity-force-lab/issues/254#issuecomment-589907780:

>Next please let me know each of the 5 above that are fixed on master, and used to have a donk but don't now.

**Comment by KatieWoe** [comment]
Donk sound seems fixed in master

**Comment by zepumph** [comment]
That's great to hear thanks! I will update GrabDragInteraction on master to only preventDefault for the `space` key case.

**Comment by zepumph** [comment]
After pairing with @KatieWoe, we found that `preventDefault()` did not prevent the "donk" sound when using space to drop the grab/drag button. I added this issue as a reference to number `#24` in https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/edit and will mark this as a won't fix and close.

First I will revert my commit above though.

Thanks @KatieWoe for some speedy iterative testing to crack this one.
